af/prio/prio.py

```diff
@@ -151,6 +160,9 @@ def generate_test_list(country_code: str, category_codes: str, limit: int):
                 "country_code": "XX" if entry["cc"] == "ZZ" else entry["cc"],
             }
         )
+    metrics.incr("total_urls_served", count=len(out))
+    metrics.incr(f"country_code_requested[cc={country_code}]")
```

Comment:
Is the `country_code` guaranteed to always be uppercase or lowercase? Do you maybe want to normalise it at the beginning?